### ISSUE TITLE
Species Component

### ISSUE THREAD

**Issue by stdavis** [agrc/roadkill-mobile#98]
Waiting on more data and specifics from DWR including #63

**Comment by nathankota** [comment]
I don't think there is an issue for this yet, but also related to the tables that support the species selection workflows, which must be generated by DWR.

**Comment by stdavis** [comment]
This would be a good issue to track the status of those tables since they will be used by this component.

**Comment by nathankota** [comment]
request for species tables sent to DWR

**Comment by nathankota** [comment]
The draft species list can be found [here](https://docs.google.com/spreadsheets/d/1jb0IREE3pPaK8sJqCKUf5IFUsw9-D_izX1grEsJ6iuo/edit#gid=0). There will definitely be more birds added to the list, but the schema will not change.

**Comment by stdavis** [comment]
What value do they want stored with the reports? I'm guessing the latin name? This would be the `species` column in the `report_infos` table.

**Comment by stdavis** [comment]
I generated a list of the most common species and am going to include the top four to use in the frequent search type:
![image](https://user-images.githubusercontent.com/1326248/135496812-d87ebe19-5518-4e28-841c-533c07ba4734.png)

Let me know if the client wants something different.

**Comment by stdavis** [comment]
@nathankota I've pushed a new version (421) to staging that contains the species component. I had to end up rebuilding the entire UX for the autocomplete because of issues with android but I think that it's much better now. There are a few outstanding questions for the client related to this:

- [ ] What value do they want stored in the species field? Currently, it stores the latin name or a combination of class/order/family/unknown depending on what values are selected.
- [ ] What species would they like to show up in the frequent list?
- [ ] What's the deal with duplicate latin names? Maybe they are modeling multiple common names for those species?

**Comment by nathankota** [comment]
Moving this back to `Specify (active)` as the client would like to make some changes, but need to meet internally to determine what those changes will be.

**Comment by nathankota** [comment]
> [ ] What species would they like to show up in the frequent list?

@stdavis are we okay to check this box, as the client was satisfied with the top three most selected species to show up here, or were there still questions about how to define the raccoon selection in the database table (i.e. northern raccoon vs. or including ringtail)?

**Comment by stdavis** [comment]
> @stdavis are we okay to check this box, as the client was satisfied with the top three most selected species to show up here, or were there still questions about how to define the raccoon selection in the database table (i.e. northern raccoon vs. or including ringtail)?

I think that we are good for now on this question. Now that it will be maintained in the database, it will be an easy fix as well.

**Comment by nathankota** [comment]
closing in favor of several newer issues that address the questions in here and expand on the functionality and design of the species component.
